scipy/stats/tests/test_stats.py

```diff
@@ -5363,10 +5365,9 @@ class Test_ttest_ind_permutations:
     b3 = [3, 4]
 
     # data for bigger test
-    np.random.seed(0)
```

Comment:
There is a rng seed on L5353 (old)/L5355 (new) ("data for most tests"). Should an rng be created on this line for use in the bigger test with the same seed as used above, i.e. `rng = np.random.default_rng(169708062)`

Reply:
It doesn't appear to matter.